Translation and New entities (Closes #185, Closes #330)

MatheusPortes opened the pull request:
# 🌐 Translation and New Entities

## Context
Some required entities for the application were missing. Additionally, most of the existing entities didn’t have translations to Portuguese, which affected the user experience for Portuguese-speaking users.

## What was done
- ✅ Added missing entities required by the application
- ✅ Added Portuguese translations for existing entities
- ✅ Fixed inconsistencies in some existing translations

## How to test
- Check if the new entities are correctly displayed in the interface
- Verify that all entities have appropriate translations where applicable

Related to improvements in the application's internationalization.

Closes #185
Closes #330

MatheusPortes committed:
tradução de dados para a personagem Amber para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem Arataki Itto para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem Arlecchino para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem Ayaka para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem Ayato para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem Baizhu para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem barbara para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem beidou para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem bennett para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem candace para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem charlotte para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem chevreuse para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem chiori para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem chongyun para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem clorinde para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem collei para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem cyno para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem dehya para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem diluc para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem diona para "pt"/pt-br

MatheusPortes committed:
settings

MatheusPortes committed:
tradução de dados para a personagem dori para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem emilie para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem eula para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem faruzan para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem fischl para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem freminet para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem furina para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem gaming para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem ganyu para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem gorou para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem hu-tao para "pt"/pt-br

MatheusPortes committed:
tradução de dados para a personagem jean para "pt"/pt-br

MatheusPortes committed:
fix for Error: Something went wrong installing the "sharp" module

MatheusPortes committed:
Merge pull request #1 from genshindev/mistress

update repository

MatheusPortes committed:
Merge remote-tracking branch 'origin/mistress' into translate

MatheusPortes committed:
adjustment in translation to "pt" for entity Albedo

MatheusPortes committed:
adjustment in translation to "pt" for entity Alhaitham

MatheusPortes committed:
adjustment in translation to "pt" for entity Aloy

MatheusPortes committed:
adjustment in translation to "pt" for entity Amber

MatheusPortes committed:
adjustment in translation to "pt" for entity Arataki Itto

MatheusPortes committed:
adjustment in translation to "pt" for entity Arlecchino

MatheusPortes committed:
adjustment in translation to "pt" for entity Ayaka

MatheusPortes committed:
adjustment in translation to "pt" for entity Ayato

MatheusPortes committed:
adjustment in translation to "pt" for entity Baizhu

MatheusPortes committed:
adjustment in translation to "pt" for entity Barbara

MatheusPortes committed:
adjustment in translation to "pt" for entity Beidou

MatheusPortes committed:
adjustment in translation to "pt" for entity Bennett

MatheusPortes committed:
adjustment in translation to "pt" for entity Candace

MatheusPortes committed:
adjustment in translation to "pt" for entity Charlotte

MatheusPortes committed:
new entity for character Chasca

MatheusPortes committed:
translation to "pt" for entity Chasca

MatheusPortes committed:
Tradução para "pt" da entidade Chevreuse

MatheusPortes committed:
Translation for "pt" of the entity Chiori

MatheusPortes committed:
Translation for "pt" of the entity Chongyun

MatheusPortes committed:
New entity for the character Citlali

MatheusPortes committed:
...

MatheusPortes committed:
Translation for "pt" of the entity Citlali

MatheusPortes committed:
Translation for "pt" of the entity Chevreuse

MatheusPortes committed:
...

MatheusPortes committed:
...

MatheusPortes committed:
Translation for "pt" of the entity Clorinde

MatheusPortes committed:
Translation for "pt" of the entity Collei

MatheusPortes committed:
Translation for "pt" of the entity Cyno

MatheusPortes committed:
Translation for "pt" of the entity Dehya

MatheusPortes committed:
Translation for "pt" of the entity Dehya

MatheusPortes committed:
Translation for "pt" of the entity Diona

MatheusPortes committed:
Translation for "pt" of the entity Dori

MatheusPortes committed:
Translation for "pt" of the entity Emilie

MatheusPortes committed:
Translation for "pt" of the entity Escoffier

MatheusPortes committed:
New entity for the character Escoffier

MatheusPortes committed:
Translation for "pt" of the entity Eula

MatheusPortes committed:
Translation for "pt" of the entity Faruzan

MatheusPortes committed:
Translation for "pt" of the entity Fischl

MatheusPortes committed:
Translation for "pt" of the entity Freminet

MatheusPortes committed:
Translation for "pt" of the entity Furina

MatheusPortes committed:
Translation for "pt" of the entity Gaming

MatheusPortes committed:
Translation for "pt" of the entity Ganyu

MatheusPortes committed:
Translation for "pt" of the entity Gorou

MatheusPortes committed:
Translation for "pt" of the entity Hutao

MatheusPortes committed:
New entity for the character Iansan

MatheusPortes committed:
Translation for "pt" of the entity Iansan

MatheusPortes committed:
Translation for "pt" of the entity Jean

MatheusPortes committed:
New entity for the character Ifa

MatheusPortes committed:
Translation for "pt" of the entity Ifa

MatheusPortes committed:
Translation for "pt" of the entity Kachina

MatheusPortes committed:
Translation for "pt" of the entity Kaeya

MatheusPortes committed:
Translation for "pt" of the entity Kaveh

MatheusPortes committed:
Translation for "pt" of the entity Kaveh

MatheusPortes committed:
Translation for "pt" of the entity Kazuha

MatheusPortes committed:
Translation for "pt" of the entity Keqing

MatheusPortes committed:
Translation for "pt" of the entity Kinich

MatheusPortes committed:
Translation for "pt" of the entity Kirara

MatheusPortes committed:
Translation for "pt" of the entity Klee

MatheusPortes committed:
Translation for "pt" of the entity Kokomi

MatheusPortes committed:
Translation for "pt" of the entity Kuki

MatheusPortes committed:
New entity for the character Lan Yan

MatheusPortes committed:
Translation for "pt" of the entity Lan Yan

MatheusPortes committed:
Translation for "pt" of the entity Layla

MatheusPortes committed:
Translation for "pt" of the entity Lisa

MatheusPortes committed:
Translation for "pt" of the entity Lynette

MatheusPortes committed:
Translation for "pt" of the entity Lyney

MatheusPortes committed:
New entity for the character Mavuika

MatheusPortes committed:
Translation for "pt" of the entity Mavuika

MatheusPortes committed:
Translation for "pt" of the entity Mika

MatheusPortes committed:
Translation for "pt" of the entity Mona

MatheusPortes committed:
Translation for "pt" of the entity Mualani

MatheusPortes committed:
Translation for "pt" of the entity Nahida

MatheusPortes committed:
Translation for "pt" of the entity Navia

MatheusPortes committed:
Translation for "pt" of the entity Nilou

MatheusPortes committed:
Translation for "pt" of the entity Neuvillette

MatheusPortes committed:
Translation for "pt" of the entity Ningguang

MatheusPortes committed:
Translation for "pt" of the entity Noelle

MatheusPortes committed:
New entity for the character Ororon

MatheusPortes committed:
Translation for "pt" of the entity Ororon

MatheusPortes committed:
Translation for "pt" of the entity Qiqi

MatheusPortes committed:
Translation for "pt" of the entity Raiden

MatheusPortes committed:
Translation for "pt" of the entity Razor

MatheusPortes committed:
Translation for "pt" of the entity Rosaria

MatheusPortes committed:
Translation for "pt" of the entity Sara

MatheusPortes committed:
Translation for "pt" of the entity Sayu

MatheusPortes committed:
Translation for "pt" of the entity Sethos

MatheusPortes committed:
Translation for "pt" of the entity Shenhe

MatheusPortes committed:
Translation for "pt" of the entity Shikanoin Heizou

MatheusPortes committed:
Translation for "pt" of the entity Sigewinne

MatheusPortes committed:
Translation for "pt" of the entity Sucrose

MatheusPortes committed:
Translation for "pt" of the entity Tartaglia

MatheusPortes committed:
Translation for "pt" of the entity Thoma

MatheusPortes committed:
Translation for "pt" of the entity Tighnari

MatheusPortes committed:
New entity for the character Varesa

MatheusPortes committed:
Translation for "pt" of the entity Varesa

MatheusPortes committed:
Translation for "pt" of the entity Venti

MatheusPortes committed:
Translation for "pt" of the entity Wanderer

MatheusPortes committed:
Translation for "pt" of the entity Wanderer

MatheusPortes committed:
Translation for "pt" of the entity Wriothesley

MatheusPortes committed:
Translation for "pt" of the entity Xiangling

MatheusPortes committed:
Translation for "pt" of the entity Xianyun

MatheusPortes committed:
Translation for "pt" of the entity Xiao

MatheusPortes committed:
New entity for the character Xilonen

MatheusPortes committed:
Translation for "pt" of the entity Xilonen

MatheusPortes committed:
Translation for "pt" of the entity Xingqiu

MatheusPortes committed:
Translation for "pt" of the entity Xinyan

MatheusPortes committed:
Translation for "pt" of the entity Yae Miko

MatheusPortes committed:
Translation for "pt" of the entity Yanfei

MatheusPortes committed:
Translation for "pt" of the entity Yaoyao

MatheusPortes committed:
Translation for "pt" of the entity Yelan

MatheusPortes committed:
Translation for "pt" of the entity Yoimiya

MatheusPortes committed:
Translation for "pt" of the entity Yunjin

MatheusPortes committed:
Translation for "pt" of the entity Zhongli

MatheusPortes committed:
New entity for the character Yumemizuki Mizuki

MatheusPortes committed:
Translation for "pt" of the entity Yumemizuki Mizuki

MatheusPortes committed:
New entity for the character traveler-pyro

MatheusPortes committed:
Translation for "pt" of the entity traveler-pyro

MatheusPortes committed:
Translation for "pt" of the entity traveler-hydro

MatheusPortes committed:
Translation for "pt" of the entity traveler-geo

MatheusPortes committed:
Translation for "pt" of the entity traveler-electro

MatheusPortes committed:
Translation for "pt" of the entity traveler-dendro

MatheusPortes committed:
Translation for "pt" of the entity traveler-anemo

MatheusPortes committed:
Merge branch 'genshindev:mistress' into mistress

MatheusPortes committed:
conflict fix

man90es commented on Dockerfile:
No changes in Dockerfile are needed, everything already works properly

man90es reviewed (changes requested):
The data looks good, but the changes in Dockerfile are out of scope of this PR and aren't needed

MatheusPortes committed:
New entity for weapon A Thousand Blazing Suns

MatheusPortes committed:
Translation for the entity A Thousand Blazing Suns

MatheusPortes committed:
Translation for the entity a thousand floating dreams

MatheusPortes committed:
New entity for weapon Absolution

MatheusPortes committed:
Translation for the entity Absolution

MatheusPortes committed:
Translation for the entity Akuoumaru

MatheusPortes committed:
Translation for the entity alley-hunter

MatheusPortes committed:
Translation for the entity amenoma-kageuchi

MatheusPortes committed:
Translation for the entity amos-bow

MatheusPortes committed:
Translation for the entity apprentice-s-notes

MatheusPortes committed:
Translation for the entity aqua-simulacra

MatheusPortes committed:
Translation for the entity aquila-favonia

MatheusPortes committed:
Translation for the entity ash-graven-drinking-horn

MatheusPortes committed:
New entity for weapon Ash-Graven Drinking Horn

MatheusPortes committed:
New entity for weapon Astral Vulture's Crimson Plumage

MatheusPortes committed:
Translation for the entity astral-vulture-s-crimson-plumage

MatheusPortes committed:
Translation for the entity ballad-of-the-boundless-blue

MatheusPortes committed:
Translation for the entity ballad-of-the-fjords

MatheusPortes committed:
Translation for the entity beacon-of-the-reed-sea

MatheusPortes committed:
Translation for the entity beginner-s-protector

MatheusPortes committed:
Translation for the entity black-tassel

MatheusPortes committed:
Fixing entity text Blackcliff Agate

MatheusPortes committed:
Entity translation blackcliff-agate

MatheusPortes committed:
Entity translation blackcliff-agate

MatheusPortes committed:
Fixing the Blackcliff Amulet entity text to Blackcliff Agate

MatheusPortes committed:
Entity translation blackcliff-longsword

MatheusPortes committed:
Entity translation blackcliff-pole

MatheusPortes committed:
Entity translation blackcliff-slasher

MatheusPortes committed:
Entity translation blackcliff-warbow

MatheusPortes committed:
Entity translation bloodtainted-greatsword

MatheusPortes committed:
Entity translation calamity-of-eshu

MatheusPortes committed:
Entity translation chain-breaker

MatheusPortes committed:
Entity translation cloudforged

MatheusPortes committed:
Fixing entity text crimson-moon-s-semblance

MatheusPortes committed:
Entity translation calamity-of-eshu

MatheusPortes committed:
Entity translation calamity-queller

MatheusPortes committed:
Entity translation cashflow-supervision

MatheusPortes committed:
Entity translation chain-breaker

MatheusPortes committed:
Entity translation cinnabar-spindle

MatheusPortes committed:
Entity translation cloudforged

MatheusPortes committed:
Entity translation compound-bow

MatheusPortes committed:
Entity translation cool-steel

MatheusPortes committed:
Entity translation crane-s-echoing-call

MatheusPortes committed:
Entity translation crescent-pike

MatheusPortes committed:
Entity translation crimson-moon-s-semblance

MatheusPortes committed:
Entity translation dark-iron-sword

MatheusPortes committed:
Entity translation deathmatch

MatheusPortes committed:
Entity translation debate-club

MatheusPortes committed:
Entity translation dialogues-of-the-desert-sages

MatheusPortes committed:
Entity translation dodoco-tales

MatheusPortes committed:
Entity translation dragon-s-bane

MatheusPortes committed:
Entity translation dragonspine-spear

MatheusPortes committed:
Entity translation dull-blade

MatheusPortes committed:
New entity for the weapon earth-shaker

MatheusPortes committed:
New entity for the weapon fang-of-the-mountain-king

MatheusPortes committed:
New entity for the weapon flower-wreathed-feathers

MatheusPortes committed:
New entity for the weapon flute-of-ezpitzal

MatheusPortes committed:
New entity for the weapon footprint-of-the-rainbow,
fruitful-hook,
mountain-bracing-bolt,
peak-patrol-song,
ring-of-yaxche,
sequence-of-solitude,
silvershower-heartstrings,
starcaller's-watch,
sturdy-bone,
sunny-morning-sleep-in,
surf's-up,
symphonist-of-scents,
tamayuratei-no-ohanashi,
tome-of-the-eternal-flow,
vivid-notions,
waveriding-whirl

MatheusPortes committed:
Fixing the Blackcliff grudge entity text to Blackcliff Starglitter

MatheusPortes committed:
Fixing the Blackcliff malice entity text to Blackcliff Amber

MatheusPortes committed:
Fixing entity text:
prospector's-drill
prototype-amber
prototype-starglitter
the-dockhand's-assistant

MatheusPortes committed:
Fixing the "Tome of the Eternal" entity text to "Tome of the Eternal Flow"

MatheusPortes committed:
Entity translation:
earth-shaker,
elegy-for-the-end,
emerald-orb,
end-of-the-line,
engulfing-lightning,
everlasting-moonglow,
eye-of-perception,
fading-twilight,
fang-of-the-mountain-king,
favonius-codex,
favonius-greatsword,
favonius-lance,
favonius-sword,
favonius-warbow,
ferrous-shadow,
festering-desire,
fillet-blade,
finale-of-the-deep,
fleuve-cendre-ferryman,
flower-wreathed-feathers,
flowing-purity,
flute-of-ezpitzal,
footprint-of-the-rainbow

MatheusPortes committed:
Entity translation:
forest-regalia,
freedom-sworn,
frostbearer,
fruit-of-fulfillment,
fruitful-hook,
hakushin-ring,
halberd,
hamayumi,
haran-geppaku-futsu,
harbinger-of-dawn,
hunter-s-bow,
hunter-s-path,
ibis-piercer,
iron-point,
iron-sting,
jadefall-s-splendor

MatheusPortes committed:
Entity translation:
kagotsurube-isshin,
kagura-s-verity,
katsuragikiri-nagamasa,
key-of-khaj-nisut,
king-s-squire,
kitain-cross-spear,
light-of-foliar-incision,
lion-s-roar,
lithic-blade,
lithic-spear,
lost-prayer-to-the-sacred-winds,
lumidouce-elegy,
luxurious-sea-lord,
magic-guide,
mailed-flower,
makhaira-aquamarine,
mappa-mare

MatheusPortes committed:
Entity translation para pt-br:
memory-of-dust,
messenger,
missive-windspear,
mistsplitter-reforged,
mitternachts-waltz,
moonpiercer,
mountain-bracing-bolt,
mouun-s-moon,
oathsworn-eye,
old-merc-s-pal,
otherworldly-story,
peak-patrol-song,
pocket-grimoire,
polar-star,
portable-power-saw,
predator,
primordial-jade-cutter,
primordial-jade-winged-spear,
prospectors-drill,
prototype-archaic,
prototype-crescent,
prototype-rancour,
rainslasher,
range-gauge,
raven-bow,
recurve-bow,
redhorn-stonethresher,
rightful-reward

MatheusPortes committed:
Entity translation para pt-br:
ring-of-yaxche,
royal-bow,
royal-greatsword,
royal-grimoire,
royal-longsword,
royal-spear,
rust,
sacrificial-bow,
sacrificial-fragments,
sacrificial-greatsword,
sacrificial-jade,
sacrificial-sword,
sapwood-blade,
scion-of-the-blazing-sun,
seasoned-hunter-s-bow,
sequence-of-solitude,
serpent-spine,
sharpshooter-s-oath,
silver-sword,
silvershower-heartstrings,
skyrider-greatsword,
skyrider-sword,
skyward-atlas,
skyward-blade,
skyward-harp,
skyward-pride,
skyward-spine,
slingshot,
snow-tombed-starsilver,
solar-pearl

MatheusPortes committed:
Entity translation para pt-br:
song-of-broken-pines,
song-of-stillness,
splendor-of-tranquil-waters,
staff-of-homa,
staff-of-the-scarlet-sands,
starcaller-s-watch,
sturdy-bone,
summit-shaper,
sunny-morning-sleep-in,
surf-s-up,
sword-of-descension,
sword-of-narzissenkreuz,
symphonist-of-scents,
talking-stick,
tamayuratei-no-ohanashi,
the-alley-flash,
the-bell,
the-black-sword,
the-catch,
the-dockhands-assistant,
the-first-great-magic,
the-flute,
the-stringless,
the-unforged,
the-viridescent-hunt,
the-widsith,
thrilling-tales-of-dragon-slayers,
thundering-pulse,
tidal-shadow,
tome-of-the-eternal-flow

MatheusPortes committed:
Entity translation para pt-br:
toukabou-shigure,
traveler-s-handy-sword,
tulaytullah-s-remembrance,
twin-nephrite,
ultimate-overlord-s-mega-magic-sword,
uraku-misugiri,
verdict,
vivid-notions,
vortex-vanquisher,
wandering-evenstar,
waster-greatsword,
wavebreaker-s-fin,
waveriding-whirl,
white-iron-greatsword,
white-tassel,
whiteblind,
windblume-ode,
wine-and-song,
wolf-fang,
wolf-s-gravestone,
xiphos-moonlight

MatheusPortes committed:
Entity translation para jp:
prototype-amber,
prototype-starglitter

MatheusPortes committed:
removal of changes made to the docker file

MatheusPortes commented:
I removed the changes made to the docker file

man90es commented on assets/data/characters/ifa/en.json:
```diff
@@ -0,0 +1,404 @@
+  "nation": "Natlan",
+  "affiliation": "Tlalocan",
+  "rarity": 4,
+  "release": "21-05-2025",
```
He was released on May 07, 2025

man90es commented on assets/data/characters/iansan/en.json:
```diff
@@ -0,0 +1,473 @@
+  "nation": "Natlan",
+  "affiliation": "Teteocan",
+  "rarity": 4,
+  "release": "30-04-2025",
```
She was released on March 26, 2025

man90es commented on assets/data/characters/lan-yan/en.json:
```diff
@@ -0,0 +1,397 @@
+  "nation": "Liyue Harbor",
+  "affiliation": "Chenyu Vale",
+  "rarity": 4,
+  "release": "22-01-2025",
```
She was released on January 21, 2025

man90es commented on assets/data/characters/mavuika/en.json:
```diff
@@ -0,0 +1,654 @@
+  "nation": "Natlan",
+  "affiliation": "Huitztlan",
+  "rarity": 5,
+  "release": "22-01-2025",
```
She was released on January 01, 2025

man90es reviewed (changes requested):
I didn't check all release dates but there are probably more wrong ones

MatheusPortes commented:
Okay,
I have the following dates as a reference for updates found in the community. Is there a place where I can officially check? reference:

    "1.0": "28-09-2020",
    "1.1": "20-10-2020",
    "1.2": "23-12-2020",
    "1.3": "02-03-2021",
    "1.4": "28-04-2021",
    "1.5": "26-05-2021",
    "1.6": "23-06-2021",

    "2.0": "01-07-2021",
    "2.1": "15-09-2021",
    "2.2": "13-10-2021",
    "2.3": "08-12-2021",
    "2.4": "19-01-2022",
    "2.5": "02-03-2022",
    "2.6": "12-04-2022",
    "2.7": "26-05-2022",
    "2.8": "15-06-2022",

    "3.0": "28-08-2022",
    "3.1": "07-09-2022",
    "3.2": "12-10-2022",
    "3.3": "23-11-2022",
    "3.4": "11-01-2023",
    "3.5": "22-02-2023",
    "3.6": "08-04-2023",
    "3.7": "24-05-2023",
    "3.8": "09-08-2023",

    "4.0": "23-08-2023",
    "4.1": "20-09-2023",
    "4.2": "25-10-2023",
    "4.3": "06-12-2023",
    "4.4": "24-01-2024",
    "4.5": "14-03-2024",
    "4.6": "30-04-2024",
    "4.7": "22-05-2024",
    "4.8": "10-07-2024",
    "4.9": "04-09-2024",

    "5.0": "18-09-2024",
    "5.1": "23-10-2024",
    "5.2": "04-12-2024",
    "5.3": "22-01-2025",
    "5.4": "12-03-2025",
    "5.5": "30-04-2025",
    "5.6": "21-05-2025",
    "5.7": "10-07-2025",
    "5.8": "04-09-2025",

man90es commented:
> Okay, I have the following dates as a reference for updates found in the community. Is there a place where I can officially check?

Some characters get released mid-version, here's the list of banners: https://genshin-impact.fandom.com/wiki/Wish/History
The earliest banner where a character appears equals its release date

MatheusPortes committed:
fixed inconsistency of character release date data for PT translations/entity

MatheusPortes committed:
New entities for:
Skirk,
Dahlia

MatheusPortes committed:
Translation for:
Dahlia,
Skirk

MatheusPortes commented:
I made the release date changes based on this page: https://genshin-impact.fandom.com/wiki/Character/List#Characters_by_Release_Date
from the same site

I took the opportunity to add entities and translations of 2 characters that were missing.
Skirk,
Dahlia
ps. the release dates have already been corrected.

man90es commented on assets/data/characters/chasca/en.json:
```diff
@@ -0,0 +1,474 @@
+  "nation": "Natlan",
+  "affiliation": "Tlalocan",
+  "rarity": 5,
+  "release": "04-12-2024",
```
Chasca was released on November 20, 2024 https://genshin-impact.fandom.com/wiki/Chasca

man90es commented on assets/data/characters/citlali/en.json:
```diff
@@ -0,0 +1,512 @@
+  "nation": "Natlan",
+  "affiliation": "Mictlan",
+  "rarity": 5,
+  "release": "2025-01-22",
```
Citlali was released on January 01, 2025 https://genshin-impact.fandom.com/wiki/Citlali

man90es commented on assets/data/characters/escoffier/en.json:
```diff
@@ -0,0 +1,453 @@
+  "nation": "Fontaine",
+  "affiliation": "Fontaine",
+  "rarity": 5,
+  "release": "21-05-2025",
```
Escoffier was released on May 07, 2025 https://genshin-impact.fandom.com/wiki/Escoffier

man90es reviewed (changes requested):
I suspect there are many more mistakes in this PR aside from the ones I indicated. Including but not limited to character release dates -- those were just the easiest for me to point out. Please don't make me point out each of them. And if we want to have an updated database sooner -- the best approach would be to have smaller, more targeted pull requests instead of one that changes over 300 files.

MatheusPortes commented:
The information was taken from the official Hoyolab website # https://wiki.hoyolab.com/pc/genshin/aggregate/2 # except for the dates that are conflicting. I will remove this information and leave only data from the official website, but I have seen incompatibility of information from the official "hoyolab" and fandom websites.
